gamification/management/commands/update_participants_data.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from gamification.process.update import (
+    get_mr_objects,
+    update_participants_data_with_mr,
+    get_issue_objects,
+    update_participants_data_with_issue,
+    get_participant_objects,
+    award_badges,
+)
+
+
+class Command(BaseCommand):
+    help = 'Update participants data'
+
+    def handle(self, *args, **options):
+        logger = logging.getLogger(__name__)
+        for mr in get_mr_objects():
+            if not mr.labels.count():
+                logger.warning(
+                    'Merge request: %s has no labels.' % mr)
+                continue
+            update_participants_data_with_mr(mr)
+        for issue in get_issue_objects():
+            if not issue.labels.count():
+                logger.warning(
+                    'Issue: %s has no labels.' % issue)
```

Comment:
please look at the log https://deploy-preview-160--coala-community.netlify.com/log/

that is really hard to investigate these warnings.  Can we have a URL in the log ?

Reply:
Yes, but then we have to create a function to generate the url from the issue and mr objects *as we don't have an url field in the Issue or MergeRequest models*.
e.g. 
```
url = 'https://' + issue.hoster.lower() +  '/' + issue.repo + '/issues/' + issue.number
```
this will work for both the hoster but for a merge_request url depend on the hoster as GitHub-->pull is change to `merge_request` on GitLab.

Though, we have a `repo` field in the models but currently `issue.repo` would return `null` But I will change the api to include `repo` key soon.